app/partials/data_sources.html

```diff
@@ -147,6 +147,13 @@
                     <p>The Europe PubMed Central (<a href="http://europepmc.org/">Europe PMC</a>) provides evidence of links between targets and diseases. They mine the titles, abstracts and full text research articles from both PubMed and PubMed Central, and use an entity-to-entity recognition approach to detect co-occurrences of target and disease entities.</p>
                 </div>
 
+                 <div class="cttv-docs-section">
```

Comment:
This should not show in the public, but be a partial that can be optionally imported imho.